Comp2024/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -210,9 +210,9 @@ private void configureButtonBindings( )
     //
     // Driver - A, B, X, Y
     m_driverPad.a( ).onTrue(new Dummy("driver A"));
-    m_driverPad.b( ).whileTrue(m_drivetrain.drivePathtoPose(m_drivetrain, VIConsts.kStageRight));        // drive to stage right
-    m_driverPad.x( ).whileTrue(m_drivetrain.drivePathtoPose(m_drivetrain, VIConsts.kStageLeft));         // drive to stage left
-    m_driverPad.y( ).whileTrue(m_drivetrain.drivePathtoPose(m_drivetrain, VIConsts.kStageCenter));       // drive to stage center
+    m_driverPad.b( ).onTrue(m_drivetrain.drivePathtoPose(m_drivetrain, VIConsts.kStageRight));        // drive to stage right
```

Comment:
I think this should be tested both as whileTrue as well as onTrue. I've see other teams code use whileTrue. Let's see if there's a difference in our implemenation.

Reply:
I think we figured out that onTrue is the likely case--resolving as closed.